libraries/mbed/targets/cmsis/TARGET_Atmel/TARGET_SAM_CortexM0P/utils/compiler.h

```diff
@@ -139,13 +139,14 @@
 #endif
 
 /* Define WEAK attribute */
-#if defined   ( __CC_ARM   )
-#   define WEAK __attribute__ ((weak))
-#elif defined ( __ICCARM__ )
-#   define WEAK __weak
-#elif defined (  __GNUC__  )
-#   define WEAK __attribute__ ((weak))
-#endif
+//defined in toochain.h
```

Comment:
Wouldn't it be cleaner to just delete these lines from the code and add a comment to the commit message which indicates that it was done because they are already defined in toolchain.h?  If anyone wants to see what was there before they can look at the git history.